package.json

```diff
@@ -17,64 +30,39 @@
       "email": "tiago.ribeiro@gmail.com"
     }
   ],
-  "homepage": "https://uphold.github.io/validator.js-asserts/",
-  "bugs": "https://github.com/uphold/validator.js-asserts/issues",
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/uphold/validator.js-asserts"
-  },
-  "main": "./dist/index.js",
-  "keywords": [
-    "validator.js",
-    "asserts",
-    "extra",
-    "extras",
-    "set"
-  ],
-  "options": {
-    "mocha": "--compilers js:babel-register --recursive test"
-  },
+  "main": "./src/index.js",
   "scripts": {
     "changelog": "github_changelog_generator --header-label='# Changelog' --future-release=v$npm_config_future_release && sed -i '' -e :a -e '$d;N;2,4ba' -e 'P;D' CHANGELOG.md",
```

Comment:
Update to `@uphold/github-changelog-generator` 😁?

Reply:
Yeah, was avoiding doing that here.. and on a new PR work on the release flow